### PR TITLE
[oceanbase] support case-insensitive column name

### PR DESCRIPTION
**Pull request by whhe**
Try to bypass  issue #2556.

### CODE REVIEW

File: docs/content/connectors/oceanbase-cdc(ZH).md [hunk 1]
```diff
@@ -311,6 +311,13 @@ OceanBase CDC 连接器包括用于 SQL 和 DataStream API 的选项，如下表
                 <td>String</td>
                 <td>传递自定义 JDBC URL 属性的选项。用户可以传递自定义属性，如 'jdbc.properties.useSSL' = 'false'。</td>
             </tr>
+            <tr>
+                <td>column.case-sensitive</td>
+                <td>否</td>
+                <td style="word-wrap: break-word;">true</td>
+                <td>String</td>
```
Comment (lvyanquan) [inline docs/content/connectors/oceanbase-cdc(ZH).md:318]: should be `Boolean` type?

**Review by lvyanquan — APPROVED** [review]

**Comment by zane-rl** [comment]
if set column.case-sensitive=false config, the column name cotains capital letter also read null, for example a column name is 'productTitle'
@whhe

**Comment by whhe** [comment]
> if set column.case-sensitive=false config, the column name cotains capital letter also read null, for example a column name is 'productTitle' @whhe

You are right, so when it is set to be false, user shoud use lowercase fileds in flink sql.

**Comment by whhe** [comment]
> if set column.case-sensitive=false config, the column name cotains capital letter also read null, for example a column name is 'productTitle' @whhe

I thought about it again and realized that the insensitive fields in flink sql should also be supported at the same time. The latest commit is made for it.

**Comment by leonardBang** [comment]
We may need consider more case-sensitive with Flink SQL‘s case-sensitive

**Pull request closed without merging**